configuration/menu.lua

```diff
@@ -0,0 +1,207 @@
+end
+
+--Modules for configuring start here
+local function infoScreen(x, y, width, height, text, title)
+    flush()
+    currentTab = gui.wrappedTextBox(x, y, width, height, text, title)
+    renderer.update()
+end
+
+local function configScreen(x, y, width, height, title, data)
+    flush()
+    currentTab = gui.configMenu(x, y, width, height, title, data)
+    graphics.context().gpu.setActiveBuffer(currentTab)
+    local configurationTab = data.configure(x, y, gui, graphics, renderer, currentTab)
+    currentTab = {currentTab}
+    if configurationTab ~= nil then for i = 1, #configurationTab do table.insert(currentTab, configurationTab[i]) end end
+end
+
+local selector = nil
+local function moduleSelector(x, y, width, height)
+    if selector ~= nil then
+        renderer.removeObject(selector)
+    end
+    local buttons = {}
+    for i = 1, #modules do
+        local onActivation =
+        {
+            {displayName = "Activate",
+            value = activate,
+            args = {modules[i].module, modules[i].name, modules[i].desc}},
+            {displayName = "Info",
+            value = infoScreen,
+            args = {location.x+2*selectionBoxWidth+2, location.y, maxWidth-(location.x+2*selectionBoxWidth+2), maxheight-5, modules[i].desc, modules[i].name}},
+            {displayName = "Configure",
+            value = configScreen,
+            args = {location.x+2*selectionBoxWidth+2, location.y, maxWidth-(location.x+2*selectionBoxWidth+2), maxheight-5, modules[i].name, require(modules[i].module)}}
+        }
+        table.insert(buttons, {name = modules[i].name, desc = modules[i].desc, func = gui.selectionBox, args = {x+width/2, y+i, onActivation}})
+    end
+    selector = gui.multiButtonList(x, y, buttons, width, height, "Available ".."("..math.floor(#buttons)..")")
+end
+moduleSelectorVar = moduleSelector
+
+local deSelector = nil
+local function moduleDeSelector(x, y, width, height)
+    if deSelector ~= nil then renderer.removeObject(deSelector) end
+    local buttons = {}
+    for i = 1, #processes do
+        local onActivation =
+        {
+            {displayName = "Deactivate",
+            value = deactivate,
+            args = {processes[i].module, processes[i].name, processes[i].desc}},
+            {displayName = "Info",
+            value = infoScreen,
+            args = {location.x+2*selectionBoxWidth+2, location.y, maxWidth-(location.x+2*selectionBoxWidth+2), maxheight-5, processes[i].desc, processes[i].name}},
+            {displayName = "Configure",
+            value = configScreen,
+            args = {location.x+2*selectionBoxWidth+2, location.y, maxWidth-(location.x+2*selectionBoxWidth+2), maxheight-5, processes[i].name, require(processes[i].module)}}
+        }
+        table.insert(buttons, {name = processes[i].name, func = gui.selectionBox, args = {x+width/2, y+i, onActivation}})
+    end
+    deSelector = gui.multiButtonList(x, y, buttons, width, height, "Active ".."("..math.floor(#buttons)..")")
```

Comment:
Oh, it says `"Active"` here instead of `"Available"`

Please, refactor. Don't repeat yourself